### DISCUSSION TITLE
New category slug needed: "gui-application"

### DISCUSSION THREAD

**Discussion by mark-summerfield** [rust-lang/crates.io#4387] [Feature Requests]
I think that a new category slug is needed: "gui-application".
For example, I've created two little GUI applications using fltk-rs (one a game, one a basic audio player), and plan to create more.
There is, after all, a category "command-line-utilities", and a whole bunch of web ones, but no corresponding one for gui applications. (The "gui" category is for tools that support gui programming, not gui programs themselves.)

**Comment by weihanglo** [comment]
Hi, I came across from [rust-lang/cargo#10506](https://github.com/rust-lang/cargo/issues/10506). There is a well-written guide in the root README.md regarding [how to add a new category]. In short, the proposal needs to provide examples fitting in that category and its difference from other existing categories. 

Here are some previous efforts you can refer to:

- https://github.com/rust-lang/crates.io/issues/4105
- https://github.com/rust-lang/crates.io/issues/1412
- https://github.com/rust-lang/crates.io/pull/1893
- https://github.com/rust-lang/crates.io/pull/2090

Disclaimer: I am not a member of crates.io team


[how to add a new category]: https://github.com/rust-lang/crates.io/blob/7844bf4c543859f351fd7d82d5a971c49e80ccec/README.md#categories
